ftp-ballerina/Ballerina.toml

```diff
@@ -4,10 +4,17 @@ name = "ftp"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
-path = "./lib/commons-vfs2-2.6.0.jar"
-groupId = "org.apache.commons"
+path = "./lib/commons-vfs2-2.2-wso2v1.jar"
+groupId = "org.wso2.org.apache.commons"
 artifactId = "commons-vfs2"
-version = "2.6.0"
+version = "2.2-wso2v1"
```

Comment:
Here also, we may need to parameterize the versions.

Reply:
Parameterized all the versions.